initiatives/2020-04-beta1-milestone.json

```diff
@@ -0,0 +1,52 @@
+[
+  {
+    "type": "sourcecred/initiativeFile",
+    "version": "0.2.0"
+  },
+  {
+    "title": "Release SourceCred beta.1",
+    "timestampIso": "2020-04-14T00:00:00.000Z",
+    "weight": { "incomplete": 0, "complete": 0 },
+    "completed": false,
+    "champions": [],
+    "dependencies": {
+      "entries": [
+        { "title": "Placeholder: Create a scaffolding for Docs" },
+        { "title": "Placeholder: Create the Initiatives plugin" },
+        { "title": "Placeholder: Create the Discord plugin" },
+        { "title": "Placeholder: Write the initial Intro section of the Docs" },
+        { "title": "Placeholder: Write the initial Concepts section of the Docs" },
+        { "title": "Placeholder: Create the Grain ledger" },
+        { "title": "Placeholder: Write the initial Guides section of the Docs" },
+        { "title": "Placeholder: Create visual language for beta Docs" },
+        { "title": "Placeholder: Quality of Life for instance maintainers initiative" },
+        { "title": "Placeholder: Create a Grain UI" },
+        { "title": "Placeholder: Write the initial Setup and Usage section of the Docs" },
+        { "title": "Placeholder: Create a data format to answer Cred questions with" },
+        { "title": "Placeholder: Beta.1 polish week initiative" }
+      ]
+    },
+    "contributions": {
+      "entries": [
+        {
+          "title": "[SourceCred beta.1](https://roamresearch.com/#/app/SourceCred/page/MJtfaaIhK) roam notes",
+          "contributors": [
+            "https://discourse.sourcecred.io/u/decentralion",
+            "https://discourse.sourcecred.io/u/lb",
+            "https://discourse.sourcecred.io/u/Beanow"
+          ],
+          "weight": null
+        },
```

Comment:
This pattern really drives home the roam shortcomings for me and worries me a fair bit.

With no version control, self-reported authors and no automation. The roam notes aren't going to be easy to flow Cred to for the mid term. Making a big adoption push for it I think may deteriorate our Cred quality substantially.

You could say that the self-reporting without version control aspect makes it a [Trust level 3](https://github.com/sourcecred/docs/blob/c9fbdee8525f88d615ff5127a925da1773ad02e4/culture/trust_levels.md#trust-levels-defined) exclusive platform, even if we created a plugin. And the overhead of manually tracking this while there isn't a plugin is practically guaranteed to miss contributions.

Reminding me of @s-ben's earlier comment on this, we may want to be a bit more cautious in our fast migration of all the things into roam.